centaurCwlRunner/src/main/scala/centaur/cwl/OutputManipulator.scala

```diff
@@ -21,11 +21,21 @@ object OutputManipulator extends Poly1 {
   def resolveOutput(jsValue: JsValue, pathBuilder: PathBuilder, mot: MyriadOutputType): Json  = {
     mot.fold(this).apply(jsValue, pathBuilder)
   }
-  
+
   private def populateFileFields(pathBuilder: PathBuilder)(obj: JsonObject): JsonObject = {
-    val objAsMap = obj.toMap
-    val path = pathBuilder.build(objAsMap("location").asString.get).get
-    
+    import mouse.all._
+
+    val path = pathBuilder.build(obj.kleisli("location").get.asString.get).get
```

Comment:
is there a potential to use `Option` here and thus omit the gets?

Reply:
yes, this is will be part of the changes that'll be coming in the next PR to fix some PAPI failures